aries-site/src/examples/foundation/accessibility/CorePrinciples.js

```diff
@@ -0,0 +1,95 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Grid,
+  Image,
+  Paragraph,
+  ResponsiveContext,
+  Text,
+  ThemeContext,
+} from 'grommet';
+
+const object = [
+  {
+    name: 'Perceivable',
+    description:
+      'Users must be able to identify the information and interface components of an application. For most users, this is done visually, but also can be accomplished through touch and sound.',
+    imageSrc: {
+      light:
+        '/foundationImages/accessibilityImages/Perceivable-visual-element.svg',
+      dark:
+        '/foundationImages/accessibilityImages/Perceivable-visual-element-invert.svg',
+    },
+    imageAlt:
+      'Two eye icons layered over one another on a split light and dark background to display contrast',
+  },
+  {
+    name: 'Operable',
+    description:
+      'Users must be able to control and navigate interface elements in an application. For example, allow for buttons to be clicked by a keyboard operation.',
+    imageSrc: {
+      light:
+        '/foundationImages/accessibilityImages/Operable-visual-element.svg',
+      dark:
+        '/foundationImages/accessibilityImages/Operable-visual-element-invert.svg',
+    },
+    imageAlt:
+      'A computer icon on a solid background with an up, left, right and down arrow icon displayed on the screen',
+  },
+  {
+    name: 'Understandable',
+    description:
+      'The content, information, and operations presented to a user must be understandable.',
+    imageSrc: {
+      light:
+        '/foundationImages/accessibilityImages/Understandable-visual-element.svg',
+      dark:
+        '/foundationImages/accessibilityImages/Understandable-visual-element-invert.svg',
+    },
+    imageAlt:
+      'A square, circle, rounded square, and pentagon connected with a green dotted line',
+  },
+  {
+    name: 'Robust',
```

Comment:
Is there any flexibility on the term 'Robust'? I wonder if 'adaptable' would be a better term to fit with the other 3 principles. Does that stray too far from the description? But then again, robust is nice for its brevity and readability. Thoughts?

Reply:
I am using the term Robust because it aligns with the WCAG and what they define as core principles for accessibility: https://www.boia.org/blog/what-are-the-four-major-categories-of-accessibility#:~:text=The%20Web%20Content%20Accessibility%20Guidelines,best%20way%20to%20achieve%20accessibility.

Reply:
we could change this but i figured it would be nice to remain aligned with them